frontend/src/pages/Graphing/components/Graph.tsx

```diff
@@ -838,7 +833,31 @@ const Graph = ({
 					maxHeight="screen"
 					key={series.join(';')} // Hacky but recharts' ResponsiveContainer has issues when this height changes so just rerender the whole thing
 					cssClass={clsx({ [style.disabled]: disabled })}
+					position="relative"
 				>
+					{loading && (
+						<Stack
+							position="absolute"
+							width="full"
+							height="full"
+							alignItems="center"
+							justifyContent="center"
+							cssClass={style.loadingOverlay}
+						>
+							<Badge
+								size="medium"
+								shape="basic"
+								variant="gray"
+								label="Loading"
+								iconStart={
+									<IconSolidLoading
+										className={loadingIcon}
+										color={vars.theme.static.content.weak}
+									/>
+								}
+							/>
+						</Stack>
+					)}
```

Comment:
noticed that this `loading` state is an overlay on top of the chart even if the chart is loaded

should we do something like blur the chart if it already has data?
![Screenshot from 2024-05-14 16-31-56](https://github.com/highlight/highlight/assets/1351531/af9d6b4c-c2a2-4328-8596-9cd69136a48a)

Reply:
yeah I'm not 100% sure - I tried w/ blur and other background effects but I thought they were kind of distracting. cc @julian-highlight for input here

Reply:
We could just reduce the opacity of everything apart from the title down to 10%. This way we're sure nothing is blocking sight

![image](https://github.com/highlight/highlight/assets/114743131/91fd7ca6-dba1-4c56-b13f-56225d0f6988)